modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -189,7 +189,7 @@ func (suite *InterchainAccountsTestSuite) TestOnChanOpenTry() {
 			counterparty := channeltypes.NewCounterparty(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 			channel = &channeltypes.Channel{
 				State:          channeltypes.TRYOPEN,
-				Ordering:       channeltypes.ORDERED,
+				Ordering:       channeltypes.UNORDERED,
```

Comment:
might be nice if these were just separate test cases. It's possible upgrades occur not moving to UNORDERED?

Reply:
sure, i can refactor these tests to have them be diff test cases